src/entities/Updates/types.ts

```diff
@@ -1,22 +1,32 @@
-export type UpdateAttributes = {
-  id: string
-  proposal_id: string
-  author?: string
-  health?: ProjectHealth
-  introduction?: string
-  highlights?: string
-  blockers?: string
-  next_steps?: string
-  additional_notes?: string
-  status: UpdateStatus
-  due_date?: Date
-  completion_date?: Date
-  created_at: Date
-  updated_at: Date
-  discourse_topic_id?: number
-  discourse_topic_slug?: string
+import { z } from 'zod'
+
+export type UpdateGeneral = {
```

Comment:
I think we should call them `GeneralUpdate` and `FinancialUpdate`, because if not they read like an action